cordapp-example/workflows-kotlin/src/test/kotlin/com/example/test/contract/IOUContractTests.kt

```diff
@@ -27,46 +27,57 @@ class IOUContractTests {
     }
 
     @Test
-    fun `transaction must have no inputs`() {
+    fun `Create - transaction must have no inputs`() {
         ledgerServices.ledger {
             transaction {
                 input(IOUContract.ID, IOUState(iouValue, miniCorp.party, megaCorp.party))
                 output(IOUContract.ID, IOUState(iouValue, miniCorp.party, megaCorp.party))
                 command(listOf(megaCorp.publicKey, miniCorp.publicKey), IOUContract.Commands.Create())
-                `fails with`("No inputs should be consumed when issuing an IOU.")
+                `fails with`("No IOUState inputs should be consumed when issuing an IOU.")
```

Comment:
To also confirm that you allow other inputs, you can do:

```kotlin
input(DummyContract, DummyState())`
output(IOUContract.ID...
command(listOf(...
verifies()
input(IOUContract.ID, IOUState...
`fails with`("No...
```

Reply:
https://github.com/DZariusz/corda-samples/pull/1/files#r406713774